tests/components/acaia/quality_scale.yaml

```diff
@@ -0,0 +1,113 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      No custom actions are defined.
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions:
+    status: exempt
+    comment: |
+      No custom actions are defined.
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: todo
+  entity-event-setup:
+    status: exempt
+    comment: |
+      No explicit event subscriptions.
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: done
+  test-before-configure: done
+  test-before-setup:
+    status: exempt
+    comment: |
+      Device is expected to be offline most of the time, but needs to connect quickly once available.
+  unique-config-entry: done
+  # Silver
+  action-exceptions:
+    status: exempt
+    comment: |
+      No custom actions are defined.
+  config-entry-unloading: done
+  docs-configuration-parameters: done
+  docs-installation-parameters: done
+  entity-unavailable: done
+  integration-owner: done
+  log-when-unavailable:
+    status: done
+    comment: |
+      Handled by coordinator.
+  parallel-updates: todo
+  reauthentication-flow:
+    status: exempt
+    comment: |
+      No authentication required.
+  test-coverage:
+    status: done
+    comment: |
+      100% test coverage.
+
+  # Gold
+  devices: done
+  diagnostics:
+    status: exempt
+    comment: |
+      Does not make sense for this integration - no extra state compared to entities.
```

Comment:
What about fetching more data like a Bluetooth advertisement or the model? I think it makes sense to have anyway, even if it's just small

Reply:
for BLE advertisement I'd have to ask actively during diagnostics (which means the device would need to be collected, so I don't believe that makes sense). Everything else (like the model) I can see on a screenshot of the configentry, so there is 0 value in a diagnostic json here for me tbh

Reply:
set to todo